src/doc/man/includes/options-lockfile-path.md

```diff
@@ -0,0 +1,12 @@
+{{#option "`--lockfile-path` _PATH_"}}
+Changes the path of the lockfile from the default (`./Cargo.lock`) to _PATH_. _PATH_ must end with 
```

Comment:
The default is `Cargo.lock` at the workspace root.

Reply:
Good catch. It was under assumption that we already running a command from the workspace root 😄